community/artifactory-ha/Chart.yaml

```diff
@@ -1,25 +1,22 @@
 apiVersion: v1
-name: artifactory-ha
-home: https://www.jfrog.com/artifactory/
-version: 0.8.3
-appVersion: 6.6.3
+appVersion: 6.9.1
 description: Universal Repository Manager supporting all major packaging formats,
   build tools and CI servers.
+home: https://www.jfrog.com/artifactory/
+icon: https://raw.githubusercontent.com/jfrog/artifactory-dcos/master/images/jfrog_med.png
 keywords:
 - artifactory
 - jfrog
-- DevOps
-- ICP
-- amd64
+- devops
+maintainers:
+- email: jainishs@jfrog.com
+  name: jainishshah17
+- email: eldada@jfrog.com
+  name: eldada
+- email: rimasm@jfrog.com
+  name: rimusz
+name: artifactory-ha
 sources:
 - https://bintray.com/jfrog/product/JFrog-Artifactory-Pro/view
 - https://github.com/jfrog/charts
-maintainers:
-- name: jainishshah17
-  email: jainishs@jfrog.com
-- name: eldada
-  email: eldada@jfrog.com
-- name: rimusz
-  email: rimasm@jfrog.com
-icon: https://raw.githubusercontent.com/jfrog/artifactory-dcos/master/images/jfrog_med.png
-tillerVersion: ">=2.7.2"
```

Comment:
Best practice is to set `kubeVersion` and `tillerVersion`; please consider.

Reply:
@bgower-ibm Done